script/cibuild

```diff
@@ -143,11 +198,7 @@ build_scheme ()
         action=build
     fi
 
-    run_xctool $sdkflag -scheme "$scheme" $action
+    run_build_tool $sdkflag -scheme "$scheme" $action
 }
 
-export -f build_scheme
-export -f run_xctool
-export -f parse_build
-
-main
+main
```

Comment:
Yeah, Working Copy did that. It also changed the Quick and Nimble submodules, so I need to review those.